helm-charts/code-annotation/values.yaml

```diff
@@ -0,0 +1,42 @@
+# Default values for landing.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+replicaCount: 1
+
+# If provided, these will be used by the deployment
+# nodeSelector: XXXXXXX
+
+image:
+  # same tag for everyone as it is how landing repo Makefile works
+  tag: master
+  pullPolicy: IfNotPresent
+  registry: docker.io/srcd
+  codeAnnotation:
+    name: code-annotation
```

Comment:
Are you planning to have multiple images for this in the future? If not, this is not necessary. It was done that way in landing as deployment is composed of multiple containers. The standard way of declaring this is using `image.repository` and `image.tag`, e.g. https://github.com/src-d/docs/blob/master/helm-charts/docs/values.yaml#L6-L7 and https://github.com/src-d/docs/blob/master/helm-charts/docs/templates/deployment.yaml#L20